index.html

```diff
@@ -824,7 +833,297 @@ <h3>
       </p>
 
     </section>
+    <p class="note">
+      See [[DID-CORE]] for normative requirements associated with the
+      <a href="https://www.w3.org/TR/did-core/#data-model">Data Model</a>.
+    </p>
+
+    <p class="note">
+      See [[DID-CORE]] for normative requirements associated with
+      <a href="https://www.w3.org/TR/did-core/#representations">Representations</a>.
+    </p>
+
+    <p>
+      The DID document data model contains “properties” (such as `id`, `verificationMethod`, and `service`), which are
+      defined in a way that is independent of a concrete representation. There are a number of known representations,
+      including JSON and JSON-LD (defined by DID Core), and CBOR (defined by a DID WG Note). Additional representations
+      (such as XML, YAML, CBOR-LD, and others) are possible and may be defined in the future. For each of these
```

Comment:
Avoid any may/should/must, even if lower case, 'cause people get confused and think you're expressing normative requirements. Alternatives include 'can', 'might', 'are expected to', 'might consider', etc.